insta-clone/src/components/Login/Login.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+
+class Login extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            username: '',
+            password: ''
+        } 
+    }
+
+    componentDidMount(){
+        
+    }
+
+    userChangeHandler = event => {
+        this.setState({
+            username: event.target.value,
+        })
+    }
+
+    pwChangeHandler = event => {
+        this.setState({
+            password: event.target.value,
+        })
+    }
+
+    submitHandler = event => {
+
+    }
+
+    render(){
+        return(
+            <form>
+                <input placeholder="username" onChange={this.userChangeHandler} value={this.state.username}></input>
```

Comment:
For onChange handler add a name that matches the state of that input value. For example `name = "username"`